scripts/pr_check.sh

```diff
@@ -0,0 +1,148 @@
+                                        ex -sc '1i|// License: Apache 2.0. See LICENSE file in root directory.' -cx $filename
+                                        fixed=$((fixed+1))
+                                   fi
+                                   if [[ $filename == *".h"* ]]; then
+                                        echo "Trying to auto-resolve...";
+                                        ex -sc '1i|/* License: Apache 2.0. See LICENSE file in root directory. */' -cx $filename
+                                        fixed=$((fixed+1))
+                                   fi
+                                   if [[ $filename == *".js"* ]]; then
+                                        echo "Trying to auto-resolve...";
+                                        ex -sc '1i|// License: Apache 2.0. See LICENSE file in root directory.' -cx $filename
+                                        fixed=$((fixed+1))
+                                   fi
+                                   if [[ $filename == *".txt"* ]]; then
+                                        echo "Trying to auto-resolve...";
+                                        ex -sc '1i|# License: Apache 2.0. See LICENSE file in root directory.' -cx $filename
+                                        fixed=$((fixed+1))
+                                   fi
+                              fi
+                         fi
+
+                         if [[ $(grep -o -P '\t' $filename | wc -l) -ne 0 ]]; then
+                              echo "[ERROR] $filename has tabs (this project is using spaces as delimiters)"
+                              ok=$((ok+1))
+
+                              if [[ $2 == *"fix"* ]]; then
+                                   echo "Trying to auto-resolve...";
+                                   sed -i.bak $'s/\t/    /g' $filename
+                                   fixed=$((fixed+1))
+                              fi
+                         fi
+
+                         if [[ $(file ${filename} | grep -o -P 'CRLF' | wc -l) -ne 0 ]]; then
+                              echo "[ERROR] $filename is using DOS line endings (this project is using Unix line-endings)"
+                              ok=$((ok+1))
+
+                              if [[ $2 == *"fix"* ]]; then
+                                   echo "Trying to auto-resolve...";
+                                   dos2unix $filename
+                                   fixed=$((fixed+1))
+                              fi
+                         fi
+                    fi
+               fi
+          fi
+     done
+}
+
+if [[ $1 == *"help"* ]]; then
+     echo Pull-Request Check tool
+     echo "Usage: (run from repo scripts directory)"
+     echo "    ./pr_check.sh [--help] [--fix]"
+     echo "    --fix    Try to auto-fix defects"
+     exit 0
+fi
+	
+cd ../realsense2_camera
+check_folder include $1
+check_folder src $1
+check_folder launch $1
```

Comment:
After my PR will be merge today we will also have a new folder called `tools`
Is they a way to iterate a call to `check_folder` on all `realsense2_camera` / `realsense2_camera_msgs` / `realsense2_description` sub folders?